controllers/topics/index.js

```diff
@@ -217,12 +245,14 @@ const followTopicV2 = async (req, res) => {
           detailTokenUser.user_id,
           name,
           detailTokenUser.is_anonymous,
-          with_system_message,
-          prevUserToken
+          topicInvitationIds,
+          prevUserToken,
+          with_system_message
         )
       );
 
       if (getSecondUserStatus) {
+        topicInvitationIds = [];
         await userTopicService.unfollowTopic(secondDetailUser.user_id, topic_id);
         data.push(
           await _afterPutTopic(
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

> Note: This review was outside of the patch, so it was mapped to the patch with the greatest overlap. Original lines [248-266]

The function `_afterPutTopic` has been modified to accept an additional parameter `topicInvitationIds`. This is used to handle the deletion of topic invitation channels when a user follows a topic.
```diff
-          with_system_message,
-          prevUserToken
+          topicInvitationIds,
+          prevUserToken,
+          with_system_message
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->